### PR TITLE
Add log4j configuration

### PR DESCRIPTION
**Pull request by Pr0methean**
Necessary to fix a warning at startup.

### CODE REVIEW

File: src/main/resources/log4j.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+<log4j:configuration xmlns:log4j="http://jakarta.apache.org/log4j/">
+    <appender name="console" class="org.apache.log4j.ConsoleAppender">
+        <param name="Target" value="System.out"/>
+        <layout class="org.apache.log4j.PatternLayout">
+            <param name="ConversionPattern" value="%-5p %c{1} - %m%n"/>
+        </layout>
+    </appender>
+
+    <root>
+        <priority value ="debug" />
```
Comment (smartboyathome) [inline src/main/resources/log4j.xml]: Since this is the "release" config, we probably should have the priority set to info.

**Review by smartboyathome — CHANGES_REQUESTED** [review]

**Comment by smartboyathome** [comment]
Also, in general, I think it would be more useful if log4j output were logged to a file rather than to the console.

File: src/main/resources/log4j.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        <appender-ref ref="console" />
+    </root>
+
+</log4j:configuration>
```
Comment (aramperes) [inline src/main/resources/log4j.xml]: Add a blank line at the end (LF) for both files.

**Review by aramperes — CHANGES_REQUESTED** [review]

**Comment by Pr0methean** [comment]
I think it's an important feature of Minecraft that errors and warnings are available on the console, even with the more stable Mojang code, so I've set the production log4j.xml to handle messages like so:

- Warning or error: file and console
- Info: file only
- Debug: ignore

**Review by smartboyathome — APPROVED** [review]

**Review by aramperes — APPROVED** [review]

**Review by mastercoms — APPROVED** [review]

**Pull request merged**